cmd/thanos/bucket.go

```diff
@@ -80,6 +80,12 @@ func registerBucketVerify(m map[string]setupFunc, root *kingpin.CmdClause, name
 		Short('i').Default(verifier.IndexIssueID, verifier.OverlappedBlocksIssueID).Strings()
 	idWhitelist := cmd.Flag("id-whitelist", "Block IDs to verify (and optionally repair) only. "+
 		"If none is specified, all blocks will be verified. Repeated field").Strings()
+	deleteDelay := modelDuration(cmd.Flag("delete-delay", "Time before a block marked for deletion is deleted from bucket. "+
```

Comment:
This sounds like bucket is doing deletes. I would remove it. All bucket commands should rather treat it differently on their own.

Reply:
bucket is not doing deletes, but it is marking blocks for delete.

Reply:
I have added an explanation for this. Kindly check if that will suffice.

Reply:
Resolving this thread as feedback was provided in this thread: https://github.com/thanos-io/thanos/pull/2136#discussion_r387916352